### PR TITLE
Pat/test pasting image and text

### PR DESCRIPTION
**Pull request by whlpatricia**
### Description

Verify that an user can "Paste" image/text in demopage using right click to copy and ctrl+v to paste

### Bugzilla bug ID

**Testrail:** https://mozilla.testrail.io/index.php?/cases/view/464474
**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1925568

### Type of change

- [x] New Test

### How does this resolve / make progress on that bug?

Completed

### Screenshots / Explanations

NA

### Comments / Concerns

NA

### CODE REVIEW

File: modules/page_base.py [hunk 1]
```diff
@@ -533,6 +533,51 @@ def context_click(
...
         return self
+    
+    def paste_to_element(
+        self, sys_platform, reference: Union[str, tuple, WebElement], labels=[]
+    ) -> Page:
+        """ Paste the copied item into the element"""
+        with self.driver.context(self.context_id):
+            el = self.fetch(reference, labels)
+            self.scroll_to_element(el)
+            self.actions.click(el).perform()
```
Comment (ben-c-at-moz) [inline modules/page_base.py:544]: Do you need to use ActionChains here? I think `el.click()` or `self.click_on(el)` would be fine

File: modules/page_base.py [hunk 2]
```diff
@@ -533,6 +533,51 @@ def context_click(
...
+            if sys_platform == "Darwin":
```
Comment (ben-c-at-moz) [inline modules/page_base.py:545]: This is fine, but you can factor out the key with

```Python
mod_key = Keys.COMMAND if sys_platform == "Darwin" else Keys.CONTROL
self.actions.key_down(mod_key)
...
```

**Review by ben-c-at-moz — APPROVED** [review]
Fine as is, but there are some potential organizational changes that could tighten it up.

File: modules/page_base.py (hunk 2 above)
Comment (whlpatricia) [reply modules/page_base.py]: changing to this as well

**Review by ben-c-at-moz — APPROVED** [review]

**Pull request merged**